src/map/script.c

```diff
@@ -15708,6 +15708,66 @@ BUILDIN(md5)
 	return true;
 }
 
+BUILDIN(swap)
+{
+	TBL_PC *sd = NULL;
+	struct script_data *data1, *data2;
+	const char *varname1, *varname2;
+	int64 uid1, uid2;
+
+	data1 = script_getdata(st,2);
+	data2 = script_getdata(st,3);
+
+	if (!data_isreference(data1) || !data_isreference(data2)) {
```

Comment:
It's a good idea to add ` || reference_toconstant(data1) || reference_toconstant(data2)` to the check (even if it appears to be useless at the moment) - see `BUILDIN(__setr)` for comparison